src/extensions/service-vendors/bandwidth/setup-and-numbers.js

```diff
@@ -237,7 +236,7 @@ export async function syncAccountNumbers(organization, options) {
       .whereIn("phone_number", nums)
       .pluck("phone_number");
     const newNums = existingNums.length
-      ? remove(nums, e => existingNums.indexOf(e) + 1)
+      ? nums.filter(e => existingNums.indexOf(e) == -1)
```

Comment:
The code intends to set `newNums` to an array of phone numbers that exist in `nums` but doesn't exist in `existingNums`. However, what the code actually did is set `newNums` to an array of phone numbers that exist in both `nums` and `existingNums`, since the [Lodash remove](https://lodash.com/docs/4.17.15#remove) function returns the array of elements which are removed from the input array.

The Lodash remove is swapped out in favor of a filter which sets `newNums` to the corrected intended value.

Reply:
Great catch and I appreciate using `filter` since it's nice built-in function and more readable to me